storage-inventory/ops/README.md

```diff
@@ -10,76 +10,101 @@ Read the entire document or jump to a relevant section below.
 - Resources:
-    - **Inventory database**: is the ledger tracking all the files storage at the site. Applications and services will access this database in parallel so it will need to have good performance, 
-especially as the content as the site grows.
-    - **Storage platform**: the actual back end storage platform. Currently, SI supports two backends: POSIX filesystem and the Swift Object Store API (e.g. CEPH Object Store)
+    - **Inventory database**: this is the ledger tracking all the files storage at the site. Applications and services will access this database in parallel so it will need to have good performance, especially as the content as the site grows.  Currently only Postgresql is supported.
+    - **Storage platform**: the actual back-end storage platform. Currently, SI supports two platforms: [POSIX filesystem](https://github.com/opencadc/storage-inventory/tree/master/cadc-storage-adapter-fs) and the [Swift Object Store API](https://github.com/opencadc/storage-inventory/tree/master/cadc-storage-adapter-swift) (e.g. CEPH Object Store)
 - Applications that run at a storage site:
-    - **tantar**: file-validate application that compares the inventory database against the back end storage.
-
-#### Supporting Infrastructure and Services
-The following external services are required for the SI deployment. Images with default implementations are available: 
-   - **proxy/ingress**: All the calls the front-end Web services need to go through a proxy/ingress that provides SSL termination and ensures that authentication 
-     headers are correctly set before being routed to the actual service. The proxy needs a public IP address and a valid SSl certificate (e.g. [Let's Encrypt](https://www.letsencrypt.org)).
-   - [`reg`](https://ws.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/reg/): Used to map service IDs to the actual URLs where the service is deployed decoupling the service from its actual location. The simplest **registry** is a static page but the **reg** container
-     included with the SI distribution allows the information to be configured on the fly.
-   - [`baldur`](https://ws.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/baldur): permissions service API using configurable rules to grant access based on resource identifiers (Artifact.uri values in the inventory data model).
-     his service is required if Authentication and Authorization (A&A) is required for the SI deployment. Most of the time, **baldur** works with a Group Management Service (GMS) and/or User Service.
+    - [`tantar`](https://github.com/opencadc/storage-inventory/tree/master/tantar): artifact-validate application that compares the inventory database against the back-end storage.
+    - [`ringhold`](https://github.com/opencadc/storage-inventory/tree/master/ringhold): artifact removal applications.  Use with caution: essentially the same as `\rm -r` on a _namespace_.
```

Comment:
ringhold allows one to supply arbitrary SQL that selects artifacts to be deleted. The nominal use case is removing a namespace, but it can also "remove artifacts older than one year" or "remove files of this type" or whatever.

also, it (currently) only removes the local copy (DeletedStorageLocationEvent) not all copies (DeletedArtifactEvent)